pycsw/core/repository.py

```diff
@@ -839,6 +841,8 @@ def setup(database, table, create_sfsql_tables=True, postgis_geometry_column='wk
         Column('distancevalue', Text, index=True),
         Column('distanceuom', Text, index=True),
         Column('wkt_geometry', Text),
+        Column('vert_extent_min', Float, index=True),
```

Comment:
These constitutes a model change (and DB migration for existing installations).  Just want to confirm that `vert_extent_min` and `vert_extent_max` are valuable queryables and columns to have in the database model.

Reply:
@tomkralidis See previous comments.